web/src/lib/components/ToastAction.svelte

```diff
@@ -22,10 +22,10 @@
   };
 </script>
 
-<ToastContainer {color}>
+<ToastContainer {color} class="shadow-sm border border-gray-200 dark:border-gray-900">
```

Comment:
this is kind of weird to me, since it only applies to like 2-3 usages of toasts, and all of the rest of them are going to have the default styles from the ui library. Why are you changing only these custom ones?

Reply:
yeah changing from the UI library is better, will redo these changes